### ISSUE TITLE
Moving Yii2 Admin to extensions folder

### ISSUE THREAD

**Issue by harlangray** [mdmsoft/yii2-admin#163]
I have the need to move the module to extensions folder from vendor folder. So my module is now extensions/mdmsoft.

in my config directory I add the alias  
   'mdmsoft/yii2-admin' =>
    array(
        'name' => 'mdmsoft/yii2-admin',
        'version' => '1.0.4.0',
        'alias' =>
        array(
            '@mdm/admin' => $extensionDir . '/mdmsoft/yii2-admin',
        ),
    ),

I have done similar thing to other modules like yii2 user module and it works.

However in this case it doesn't work. I get the error

"Class mdm\admin\components\AccessControl does not exist"

It works well when it is in the vendor folder. But I need to move it to extensions folder to make some changes.

**Comment by mdmunir** [comment]
good luck

**Comment by harlangray** [comment]
What do you mean good luck? Is there no answer to this? Is it not possible?

**Comment by mdmunir** [comment]
I mean i am unsure what are you doing. can you explain me what is this:

'mdmsoft/yii2-admin' =>
array(
'name' => 'mdmsoft/yii2-admin',
'version' => '1.0.4.0',
'alias' =>
array(
'@mdm/admin' => $extensionDir . '/mdmsoft/yii2-admin',
),
),

If you want to add path alias, you can do simple
```php
return[
    'vendorPath' => ...,
    'aliases' => [
        'mdm/admin' => $extensionDir . '/mdmsoft/yii2-admin',
    ],
    'components' => [
       ...
    ],
];
```

You also have error trace/error log. You can find out what heppen from there. Do your self.

**Comment by harlangray** [comment]
Yes when I added alias that way it worked. Thank you very much. I used 
'mdmsoft/yii2-admin' =>
array(
'name' => 'mdmsoft/yii2-admin',
'version' => '1.0.4.0',
'alias' =>
array(
'@mdm/admin' => $extensionDir . '/mdmsoft/yii2-admin',
),
),
(taken from auto generated extension file inside vendor/yiisoft)
in an extension file and added it to config like this

    'extensions' => array_merge(
            require(__DIR__ . '/extensions.php'), require($vendorDir . '/yiisoft/extensions.php')
    ),

This worked for yii2 user and some other modules. Not sure why it didn't work here.

Any way your idea was helpful and good to know that. Thank you.
